suite-native/module-trading/src/components/general/AccountSheet/AccountSheetHeader.tsx

```
@@ -0,0 +1,50 @@
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { SearchableSheetHeader } from '../SearchableSheetHeader';
+import { SheetHeaderTitle } from '../SheetHeaderTitle';
+
+export type AccountSheetHeaderProps = {
+    onClose: () => void;
+    selectedAccount: undefined | Account;
+    clearSelectedAccount: () => void;
+};
+
+const wrapperStyle = prepareNativeStyle<{}>(({ spacings }) => ({
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)